src/pages/workspaces/[workspaceSlug]/pipelines/[pipelineCode]/index.tsx

```diff
@@ -235,6 +244,39 @@ const WorkspacePipelinePage: NextPageWithLayout = (props: Props) => {
                 </p>
               )}
             </DataCard.FormSection>
+            {isWebhookFeatureEnabled ? (
+              <DataCard.FormSection title={t("Webhook")} onSave={onSaveWebhook}>
+                <p>
+                  <span>
+                    You can use a webhook to trigger this pipeline from an
+                    external system using a POST request. Be careful it can be
+                    triggered by anyone with the URL.
```

Comment:
Shouldn't this be translatable?

I would add a little warning sign, and also state that this is an experimental feature.

"Webhooks are experimental and don't require any form of authentication for now: anyone with the URL will be able to trigger this pipeline".